src/model/Deck.ts

```diff
@@ -0,0 +1,14 @@
+import { Model } from "@nozbe/watermelondb";
+import { date, field } from "@nozbe/watermelondb/decorators";
+import { Associations } from "@nozbe/watermelondb/Model";
+import { TableName } from "./constants";
+
+export default class Deck extends Model {
+  static table = TableName.DECKS;
+  static associations: Associations = {
+    [TableName.CARDS]: { type: "has_many", foreignKey: "deck_id" },
+  };
+
+  @field("name") name;
+  @date("created_at") created_at;
+}
```

Comment:
```suggestion
  @children("cards") cards;
}
```
Might be useful to have

Reply:
Changed in https://github.com/Dara2004/Goi/pull/51/commits/bc9449ef21e1d872f37f908ef91f6f0d7b06e7c0